hoomd/md/pytest/test_potential.py

```diff
@@ -128,10 +128,10 @@ def _make_invalid_param_dict(valid_dict):
         # Set one invalid argument per dictionary
         # Set two invalid arguments per key
         valid_value = invalid_dicts[count][key]
-        if not isinstance(valid_value, list):
+        if not (isinstance(valid_value, (list, np.ndarray))):
```

Comment:
The parenthesis around `isinstance` are unnecessary.
```suggestion
        if not isinstance(valid_value, (list, np.ndarray)):
```